docs/ext/commands/slash_commands.rst

```diff
@@ -343,78 +376,7 @@ You can implement double nesting and build commands like ``/parent group subcmd`
 .. note ::
     This is the deepest nesting available.
 
-Docstrings
-----------
-
-If you have a feeling that option descriptions make the parameters of your function look overloaded, use docstrings.
-This feature allows to describe your command and options in triple quotes inside the function, following the RST markdown.
-
-In order to describe the parameters, list them under the ``Parameters`` header, underlined with dashes:
-
-.. code-block:: python3
-
-    @bot.slash_command()
-    async def give_cookies(
-        inter: disnake.ApplicationCommandInteraction,
-        user: disnake.User,
-        amount: int = 1
-    ):
-        """
-        Give several cookies to a user
-
-        Parameters
-        ----------
-        user: The user to give cookies to
-        amount: The amount of cookies to give
-        """
-        ...
-
-.. note ::
-    In the above example we're using a simplified RST markdown.
-
-If you prefer the real RST format, you can still use it:
-
-.. code-block:: python3
-
-    @bot.slash_command()
-    async def give_cookies(
-        inter: disnake.ApplicationCommandInteraction,
-        user: disnake.User,
-        amount: int = 1
-    ):
-        """
-        Give several cookies to a user
-
-        Parameters
-        ----------
-        user: :class:`disnake.User`
-            The user to give cookies to
-        amount: :class:`int`
-            The amount of cookies to give
-        """
-        ...
-
-Specifying the types isn't necessary, the docstring will still work:
-
-.. code-block:: python3
+Injections
+++++++++++
 
-    @bot.slash_command()
-    async def give_cookies(
-        inter: disnake.ApplicationCommandInteraction,
-        user: disnake.User,
-        amount: int = 1
-    ):
-        """
-        Give several cookies to a user
-
-        Parameters
-        ----------
-        user:
-            The user to give cookies to
-        amount:
-            The amount of cookies to give
-        """
-        ...
-
-.. note ::
-    You can remove ``:`` after the parameter names if you prefer the numpy format.
+We have em, look at https://github.com/DisnakeDev/disnake/blob/master/examples/slash_commands/injections.py for more info ✨s
```

Comment:
```suggestion
We have them, look at [this example](https://github.com/DisnakeDev/disnake/blob/master/examples/slash_commands/injections.py) for more information ✨s
```

I think that using `[]()` is quite a better way than only the link. Also, why shorten words? In my opinion, we should be more polite in the documentation.

Reply:
It's just a placeholder, I meant to force eq to write docs for em later.

Reply:
Oh, alright